search-java/src/main/java/uk/os/elements/search/SearchManager.java

```diff
@@ -190,4 +242,24 @@ public SearchBundle call(SearchBundle searchBundle, List<SearchResult> localMatc
         }
         return dest;
     }
+
+    private static AddressApi provideAddressSearchApi() {
+        Retrofit retrofit = new Retrofit.Builder()
+                // TODO
+                .baseUrl("https://api.ordnancesurvey.co.uk/places/v1/addresses/")
+                .addConverterFactory(GsonConverterFactory.create())
+                .addCallAdapterFactory(RxJavaCallAdapterFactory.create())
+                .build();
+        return retrofit.create(AddressApi.class);
+    }
+
+    private static SearchApi provideSearchApi() {
+        Retrofit retrofit = new Retrofit.Builder()
+                .baseUrl("https://api.ordnancesurvey.co.uk/opennames/v1/")
+                .addConverterFactory(GsonConverterFactory.create())
+                .addCallAdapterFactory(RxJavaCallAdapterFactory.create())
+                .build();
+        return retrofit.create(SearchApi.class);
+    }
```

Comment:
@snodnipper putting these in the providers they belong to makes sense. Loss of some convenience but better separation of concerns imo.